pkgs/by-name/gh/gh-f/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, stdenvNoCC
+, makeWrapper
+, fzf
+, coreutils
+, bat
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gh-f";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "gennaro-tedesco";
+    repo = "gh-f";
+    rev = "v${version}";
+    hash = "sha256-ITl8T8Oe21m047ygFlxWVjzUYPG4rlcTjfSpsropYJw=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    install -D -m744 "gh-f" "$out/bin/gh-f"
+  '';
+
+  postInstall = ''
+    wrapProgram "$out/bin/gh-f" --prefix PATH: "${lib.makeBinPath [fzf bat coreutils]}"
```

```suggestion
    wrapProgram "$out/bin/gh-f" --prefix PATH : "${lib.makeBinPath [fzf bat coreutils]}"
```

This command wasn't successfully executed?

```
=> tree ./result/ -a
./result/
└── bin
    └── gh-f
```

---

`gh-f` worked fine but was indeed not wrapped.